docs/misc/deprecated.md

```diff
@@ -15,6 +15,10 @@ The following list of features are deprecated.
 ### Old Command Line Options
 **Deprecated In Release: [v1.8.0](/release-notes/#docker-engine-1-8-0)**
 
+The flags `-d` and `--daemon` are deprecated in favor of the `daemon` subcommand:
```

Comment:
Can you move these to be after the  "Target for removal..." line so that both headers are next to each other?